Tests/AudioKitTests/Node Tests/Generator Tests/AudioPlayer2Tests+Realtime.swift

```diff
@@ -0,0 +1,198 @@
+import AudioKit
+import AVFoundation
+import CAudioKit
+import XCTest
+
+// Real time functions, for local testing only.
+// These simulate a user interacting with the player via an UI
+// Real time functions, for local testing only
+
+// Thse are organized like this so they're easy to comment out
```

Comment:
I think you can disable individual tests via Xcode.

Reply:
oh, so my thought there was avoiding having CI do those tests. is there a way to control that?

Reply:
I was thinking this:

<img width="521" alt="image" src="https://user-images.githubusercontent.com/841144/98426554-2548d480-204e-11eb-8a5a-1e6a06d301e1.png">

Reply:
Where does it save this info if we no longer have an Xcode project file?